protected/humhub/models/forms/CropProfileImage.php

```diff
@@ -79,7 +79,7 @@ class CropProfileImage extends Model
     /**
      * @var boolean
      */
-    public $keySupport = false;
+    public $keySupport = true;
```

Comment:
Part of this PR?

Reply:
Sorry, this was added while mergin, but this is aligned with current master.